lib/tools/adb-commands.js

```diff
@@ -757,7 +790,7 @@ methods.setGeoLocation = async function (location, isEmulator = false) {
     this.adbExec(['emu', 'geo', 'fix', longitude.replace('.', ','), latitude.replace('.', ',')]);
   } else {
     return await this.shell(['am', 'startservice', '-e', 'longitude', longitude,
-                             '-e', 'latitude', latitude, `${SETTINGS_HELPER_ID}/.LocationService`]);
+                             '-e', 'latitude', latitude, `${SETTINGS_HELPER_ID}/${LOCATION_SERVICE}`]);
```

Comment:
here we might also write the whole `${SETTINGS_HELPER_ID}/${LOCATION_SERVICE}` string into a constant